pram

```diff
@@ -36,6 +36,9 @@ print_help() {
 	echo "                   GitHub repo to use (default: gentoo/gentoo)"
 	echo "  -s, --signoff    Add Signed-off-by to commits (the default)"
 	echo "  -S, --no-signoff Disable adding Signed-off-by to commits"
+	echo "  -t TOKEN, --token TOKEN"
```

Comment:
I'm not particularly eager to have people passing secrets over argv. I have a `~/.github-token` file, so perhaps we could use that.

Reply:
This is something I do personally as well. Just wasn't sure about the accepted practises elsewhere. So I'm in favour of .github-token

I still think there should be some option to make it use the api path. So maybe --api?

Reply:
Sorry, I don't understand. You mean option to pass the token explicitly? Sure, you can leave it, just note that it's taken by default from the file.

Reply:
I modified the pr to just be y/n option --api which then uses the api urls and checks uses .github-token with no ability for the user to feed it from  argv.